scanner/updater/manual/normalizeSeverity.go

```diff
@@ -0,0 +1,25 @@
+package manual
+
+import "github.com/quay/claircore"
+
+const (
+	Low       = "Low"
+	Medium    = "Medium"
+	Important = "Important"
+	Critical  = "Critical"
```

Comment:
Maybe parse the string directly in the `switch`? I don't see why we should expose these constants.